docs/pr-review-guidelines.md

```
@@ -0,0 +1,80 @@
+
+- [ ] **Verified the React & TypeScript gotchas to look out for:**
+  - [ ] `as` is a bad keyword – this likely means you're violating TypeScript's assistance and thus probably have another error in your types that you are covering up
+  - [ ] Optional chaining values accessed have fallback values
+  - [ ] Has EitherNotBoth / EitherOrNone been considered when object props are in conflict?
+  - [ ] Optional type properties are optional because they are truly optional (can optionally be passed / have a nice fallback) and not because it's easier for types (to get around a type error)
+  - [ ] Components are properly broken down into good bite-sized efforts with 1 single goal
+  - [ ] Components make liberal use of custom hooks** to store data and logic that is heavily coupled
+  - [ ] Components avoid using useMemo for processes that are not computationally expensive
+  - [ ] Referential stability has been maintained for variables through render loops and the use-cases has been considered for when it has changed
+
+     - [ ] *All functions passed to another component have referential integrity* (exception being to PF components and onChange-esk handlers)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 4; Actual: 5

(MD007, ul-indent)